src/temp/ProjectInfo/InfoContainer.tsx

```diff
@@ -0,0 +1,143 @@
+    { id: 7, title: ready ? t('common:august') : '' },
+    { id: 8, title: ready ? t('common:september') : '' },
+    { id: 9, title: ready ? t('common:october') : '' },
+    { id: 10, title: ready ? t('common:november') : '' },
+    { id: 11, title: ready ? t('common:december') : '' },
+  ];
+
+  return (
+    <div className={styles.projectInfoContainer}>
+      <div className={styles.singleInfo}>
+        <div className={styles.halfInfo}>
+          <div className={styles.infoTitle}>
+            {t('manageProjects:abandonment')}
+            <InfoIconPopup
+              height={10}
+              width={10}
+              color={`${'rgba(var(--secondary-divider-color-new))'}`}
+            >
+              <div className={styles.infoIconPopupContainer}>
+                {t('manageProjects:yearAbandonedInfo')}
+              </div>
+            </InfoIconPopup>
+          </div>
+          <div className={styles.infoDetail}>
+            {' '}
+            {t('common:approx')} {abandonment}
+          </div>
+        </div>
+        <div className={styles.halfInfo}>
+          <div className={styles.infoTitle}>
+            {t('projectDetails:firstTreePlanted')}
+          </div>
+          <time className={styles.infoDetail}>
+            {formatDate(
+              firstTree.split('-')[1].length === 1 ||
+                firstTree.split('-')[2].length === 1
+                ? addZeroToDate(firstTree)
+                : firstTree
+            )}
+          </time>
+        </div>
+      </div>
+      <div className={styles.seperator}></div>
+      <div className={styles.singleInfo}>
+        <div className={styles.halfInfo}>
+          <div className={styles.infoTitle}>
+            {' '}
+            {t('manageProjects:plantingDensity')}
+          </div>
+          <div className={styles.infoDetail}>
+            {plantingDensity}
+            {maxPlantingDensity !== null
+              ? `-${maxPlantingDensity} ${t('manageProjects:treePerHa')}`
+              : ` ${t('manageProjects:treePerHa')}`}
+          </div>
+        </div>
+        <div className={styles.halfInfo}>
+          <div className={styles.infoTitle}>
+            {t('manageProjects:employees')}
+          </div>
+          <div className={styles.infoDetail}>{employees}</div>
+        </div>
+      </div>
+      <div className={styles.seperator}></div>
+      <div className={styles.singleInfo}>
+        <div className={styles.halfInfo}>
+          <div className={styles.infoTitle}>
+            {t('projectDetails:plantingSeasons')}
+          </div>
+          <div className={styles.infoDetail}>
+            {plantingSeasons.map((season, index) => (
+              <Fragment key={seasons[season - 1].title}>
+                {seasons[season - 1].title}
+                {index === plantingSeasons.length - 2 ? (
+                  <> {t('manageProjects:and')} </>
+                ) : index === plantingSeasons.length - 1 ? (
+                  '.'
```

Comment:
{plantingSeasons.map(...)}. It may be beneficial to extract this logic into separate functions or components to improve readability.